packages/backend/src/accounting/errors.ts

```diff
@@ -1,12 +1,10 @@
-import {
-  CommitTransferError as CommitTransferErrorCode,
-  CreateTransferError as CreateTransferErrorCode
-} from 'tigerbeetle-node'
+import { CreateTransferError as CreateTransferErrorCode } from 'tigerbeetle-node'
 
 import { AccountId } from './utils'
+import { CreateAccountError as CreateAccountErrorCode } from 'tigerbeetle-node'
 
 export class CreateAccountError extends Error {
-  constructor(public code: number) {
+  constructor(public codes: number[]) {
```

Comment:
Why multiple codes?

Reply:
@sentientwaffle  The `createAccounts` function takes in an array of `accounts` to create. I wanted to also have all of those creation errors returned as part of the thrown `CreateAccountError`. That way you can get a complete list of failures when having to create a lot of accounts. But it breaks the design, so I am reverting it.